app/views/events/finance.html.erb

```diff
@@ -13,7 +13,17 @@
         </tr>
         <% ed.timecard_entries.each do |entry| %>
           <tr>
-            <td><%= entry.member %> - <%= entry.eventpart.text %></td>
+            <td><%= entry.member %>
+                <% event_roles = @event.run_positions_for(entry.member) %>
+                <% ed_roles = ed.run_positions_for(entry.member) %>
+                <% if not event_roles.empty? and not ed_roles.empty? %>
+                  (<b><% event_roles.each do |pos| %><%= pos.try(:role) %><% end %></b>, 
```

Comment:
If someone has multiple roles, does this concatenate them without punctuation or spaces? Instead of looping with `each` you might try `to_sentence`.

Reply:
Please use `<strong>` instead of `<b>`.